Chess/game.h

```diff
@@ -33,7 +32,23 @@ public slots:
     void rollback_from_list();
     void make_move_from_list();
 
+public:
+    void cellAction_impl(QString);
+
+    void startAction_impl();
+    void stopAction_impl();
+
+    void saveAction_impl(QString);
+    bool loadAction_impl(QString);
+
+    void rollback_from_list_impl();
+    void make_move_from_list_impl();
+
+    void ChangeState(State*);

```

Comment:
It starts from capital letter, all other methods start from small letter :) Should be consistent